.eslintrc.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     'react-app',
     'plugin:react/recommended',
     'plugin:@typescript-eslint/recommended',
+    'plugin:import/typescript',
```

Comment:
this adds tsx and jsx to the resolved imports so their filenames don't need to be specified in imports